descriptive-link

```diff
@@ -0,0 +1,180 @@
+---
+id: 
+name: Link text is descriptive
+rule_type: atomic
+
+description: | 
+ This rule checks that all links describe their purpose
+
+accessibility_requirements: 
+  wcag20:2.4.4: # Link Purpose (In Context) 
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+
+input_aspects:
+- DOM Tree
+
+authors:
+- Carlos Duarte
+- Marie Trudelle 
+- Ramit Garg
+---
+
+## Applicability
+
+The rule applies to any HTML element with the [semantic role](#semantic-role) of `link` that has a [non-empty](#non-empty) [accessible-name](#accessible-name).
+
+## Expectation 1
+
+Link text together with its [context](#context), if visible, describes the purpose of the link
```

Comment:
Should this be "Accessible name ... described the purpose" instead of "Link text..."? At least it needs to match the applicability, that talks about "accessible name".
Maybe it would make sense to have something both for the accessible name and the visible text of the link? Similar to what we have been trying to for https://github.com/act-rules/act-rules.github.io/pull/444/files (that is currently in debate about whether to split up the rule or keep it as one).

Reply:
@annethyme, I changed to a single expectation (and added a note) trying to cover both text content and accessible name.